### PR TITLE
Adding modules

### PR DESCRIPTION
**Pull request by hieuhho**
added reviews and info section modules

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[submodule "info-section"]
```
Comment (carlitoswillis) [inline .gitmodules:1]: very interesting

File: public/index.html [hunk 2]
```diff
@@ -2,8 +2,15 @@
...
     <head>
       <title>FEC Proxy</title>
+      <link rel="stylesheet" href="http://localhost:3002/style.css">
+      <link rel="stylesheet" href="http://localhost:3003/styles.css">
     </head>
     <body>
-      hello world
+      <div id='info-section'></div>
+      <div id='reviews'></div>
+      <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```
Comment (carlitoswillis) [inline public/index.html]: idk if this is needed since we are using jsx

**Review by carlitoswillis — APPROVED** [review]
looks good

File: public/index.html (hunk 2 above)
Comment (hieuhho) [reply public/index.html]: hmm i'll remove and test, will send another PR if working

**Review by carlitoswillis — APPROVED** [review]

**Pull request merged**